workflow/overview.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: "Workflow Builder Overview"
+sidebarTitle: "Overview"
+---
+
+Our browser-based **Workflow Builder (Editor)** makes it simple to create personalized, automated messaging workflows—no coding required.
+
+## Workflow Elements / Building Blocks
+
+All Novu workflows consist of four basic elements:
+
+- **Workflow Identifier** which is used as the trigger id 
+- A **trigger** that starts the workflow
+- **Channel Step** that send notifications to your configured channels
+- **Action Steps** that control the flow of the workflow and produce state for use in templates 
+
+### Workflow Identifier (Settings)
+
+Give your Workflow a clear, descriptive name that your team can easily identify. The workflow identifier "new-signup" is used as the trigger id. Workflow identifiers should be unique to your application and descriptive of the workflow's purpose.
+
+For example:
+
+- Abandoned Cart Recovery
+- Password Reset Flow
+- New User Welcome Series
+- Re-engagement Campaign
+
+### Trigger
+
+Every workflow starts with a **trigger**. This is the event that starts a workflow. It represents an action or event that initiates the workflow by making a call to the Novu API, along with a specified workflow trigger identifier (`workflow_id`). The identifier helps match the trigger to the corresponding workflow. 
```

Comment:
The first 2 sentences feel almost identical, maybe rephrase it?

Reply:
Addressed.